src/pages/community-page.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Link, graphql } from 'gatsby';
+// import dompurify from 'dompurify';
+import { CommunityPage } from '../types';
+import { ReleaseData } from '../hooks/useReleaseHistory';
+import Layout from '../components/Layout';
+import '../styles/community-page.scss';
+
+interface Props {
+  release?: ReleaseData;
+}
+
+interface CommunityProps {
+  data: CommunityPage;
+}
+export default function DownloadPage({ data }: CommunityProps): JSX.Element {
+  const title = 'Community Page';
+  const description = 'Community page';
+  const commCommData = data.page.html;
+  // Commented out temporarly
+  // const sanitizer = dompurify.sanitize;
+  return (
+    <Layout title={title} description={description}>
+      <main>
+        <div className="container">
+          <div className="community-header community">
+            <CommmunityHeader />
+          </div>
+          <div className="sidebar">
+            <ul>
+              <li className="community-links">
+                <Link to="/">Link 1</Link>
+              </li>
+              <li className="community-links">
+                <Link to="/">Link 2</Link>
+              </li>
+              <li className="community-links">
+                <Link to="/">Link 3</Link>
+              </li>
+              <li className="community-links">
+                <Link to="/">Link 4</Link>
+              </li>
+              <li className="community-links">
+                <Link to="/">Link 5</Link>
+              </li>
+            </ul>
+          </div>
+          <div className="community-content">
+            <div dangerouslySetInnerHTML={{ __html: commCommData }} />
+          </div>
+        </div>
+      </main>
+    </Layout>
+  );
+}
+
+const CommmunityHeader = ({ release }: Props): JSX.Element => {
+  const nodev = release?.version;
+  const npmv = release?.npm;
+  const lts = release?.lts;
+
+  return (
+    <>
+      <div className="community-page__navigation">
```

Comment:
This navigation is a redundant part of different pages. I recommend creating a component with a single CSS file for that component which can be shared by all pages.